tutor_webhook_receiver/plugin.py

```diff
@@ -2,14 +2,44 @@
 import os
 import pkg_resources
 
+from .__about__ import __version__
 
 templates = pkg_resources.resource_filename(
     "tutor_webhook_receiver", "templates"
 )
 
-config = {}
+config = {
+    "defaults": {
+        "VERSION": __version__,
+        "DOCKER_IMAGE": "webhook_receiver/webhook_receiver:latest",
+        "HOST": "webhooks.{{ LMS_HOST }}",
+        "DB_ENGINE": "django.db.backends.mysql",
+        "DB_NAME": "webhook_receiver",
+        "DB_USER": "webhook_receiver01",
+        "DB_PASSWORD": "",
+        "DB_HOST": "{{ MYSQL_HOST }}",
+        "DB_PORT": "{{ MYSQL_PORT }}",
+        "DB_MIGRATION_OPTIONS": {},
+        "DJANGO_SECRET_KEY": "",
+        "DJANGO_LOG_LEVEL": "DEBUG",
+        "LMS_BASE_URL": "",
```

Comment:
I think you can set a sane default here, based on `{{ ENABLE_HTTPS }} and {{ LMS_HOST }}`. This shouldn't be mandatory to set in `config.yml`.

Reply:
[Here's what Tutor itself does](https://github.com/overhangio/tutor/blob/master/tutor/templates/apps/openedx/config/lms.env.json#L27) to set this. Come to think of it, we might want to rename this environment variable `LMS_ROOT_URL` to be better in line with Open edX.

Reply:
OK I'll rename it and set it using `LMS_HOST`.